app/lib/backend_api_logic/service_extension.rb

```diff
@@ -18,27 +18,24 @@ module ServiceExtension
     end
 
     def first_backend_api
-      @backend_api ||= find_or_create_first_backend_api!
+      @backend_api ||= find_first_backend_api!
     end
 
-    def find_or_create_first_backend_api!
-      return unless account
-      config = backend_api_configs.first || create_first_backend_api_config!
-      config.backend_api
+    def find_first_backend_api!
```

Comment:
This shouldn't be a bang method anymore considering its content is `backend_api_configs.first&.backend_api` 😄 
```suggestion
    def find_first_backend_api
```

Reply:
In fact we don't even need this method 🤔 we could do in the method above:
```
def first_backend_api
  @backend_api ||= backend_api_configs.first&.backend_api
```